<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>753</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>956</width>
    <height>753</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>956</width>
    <height>753</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="tableInfoWidget" name="tableContain" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>771</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>9</y>
      <width>151</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="table">
       <property name="text">
        <string>테이블 주문</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="acc">
       <property name="text">
        <string>매출/통계</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="menu">
       <property name="text">
        <string>메뉴 추가 수정</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="qr">
       <property name="text">
        <string>QR 생성</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="accWidget" name="accContain" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>771</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="menuWidget" name="menuContain" native="true">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>10</y>
      <width>771</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="qrWidget" name="qrContain" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>10</y>
      <width>771</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>956</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuencore_POS">
    <property name="title">
     <string>encore-POS</string>
    </property>
    <addaction name="actionstore_user_add"/>
    <addaction name="actionGenerate_Certificate"/>
   </widget>
   <addaction name="menuencore_POS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionstore_user_add">
   <property name="text">
    <string>점포 유저 생성 </string>
   </property>
  </action>
  <action name="actionGenerate_Certificate">
   <property name="text">
    <string>인증서 생성 </string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tableInfoWidget</class>
   <extends>QWidget</extends>
   <header>table_info</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>accWidget</class>
   <extends>QWidget</extends>
   <header>acc</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>menuWidget</class>
   <extends>QWidget</extends>
   <header>menu</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>qrWidget</class>
   <extends>QWidget</extends>
   <header>qr</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
